package.json

```
@@ -3,10 +3,13 @@
   "version": "0.0.1",
   "description": "GenLayer documentation",
   "scripts": {
-    "dev": "node scripts/generate-full-docs.js && next dev",
-    "build": "node scripts/generate-full-docs.js && next build",
+    "dev": "node scripts/generate-changelog.js && node scripts/update-setup-guide-versions.js && node scripts/generate-api-docs.js && node scripts/generate-full-docs.js && next dev",
+    "build": "node scripts/generate-changelog.js && node scripts/update-setup-guide-versions.js && node scripts/generate-api-docs.js && node scripts/generate-full-docs.js && next build",
     "start": "next start",
-    "generate-sitemap": "node scripts/generate-sitemap-xml.js"
+    "generate-sitemap": "node scripts/generate-sitemap-xml.js",
+    "generate-changelog": "node scripts/generate-changelog.js",
+    "generate-api-docs": "node scripts/generate-api-docs.js",
+    "update-setup-guide": "node scripts/update-setup-guide-versions.js"
```

these should probably be 
```suggestion
    "node-generate-changelog": "node scripts/generate-changelog.js",
    "node-generate-api-docs": "node scripts/generate-api-docs.js",
    "node-update-setup-guide": "node scripts/update-setup-guide-versions.js"
```
since this repo contains many docs